cli/kaos_cli/commands/build.py

```diff
@@ -122,6 +136,60 @@ def build(backend: BackendFacade, cloud, env, force, verbose, yes, local_backend
         handle_exception(e)
 
 
+# build list
+# =============
+@build.command(name='list', short_help='List currently deployed backend builds')
+@pass_obj(BackendFacade)
+def list_all(backend: BackendFacade):
+    """
+    List all deployed kaos backend infrastructures.
+    """
+    try:
+        available_contexts = backend.list()
+        if len(available_contexts) > 0:
+            backend.cache(available_contexts)
+            table = render_table(available_contexts, include_ind=False)
+            click.echo(table)
+
+    except Exception as e:
+        handle_specific_exception(e)
+        handle_exception(e)
+
+
+# build set
+# =============
+@build.command(name='set', short_help='Set active build context')
+@click.option('-c', '--context', type=str, help='set to a specific deployed context', cls=NotRequiredIf,
+              not_required_if='ind')
+@click.option('-i', '--ind', type=str, help='set to a specific deployed index', cls=NotRequiredIf,
+              not_required_if='context')
+@pass_obj(BackendFacade)
+def set_active_context(backend: BackendFacade, context, ind):
```

Comment:
If the `context` and `ind` are not mandatory arguments, then please type-annotate them appropriately, i.e.: `... context: Optional[str] = None, ind: Optional[str] = None`

Reply:
Valid point. I have included this change in this place for now. However, this needs to be done across all modules in order to maintain uniformity (Maybe under a refactor branch?)